firebase-sessions/firebase-sessions.gradle.kts

```diff
@@ -26,7 +26,7 @@ firebaseLibrary {
 
 android {
   val targetSdkVersion: Int by rootProject
-  compileSdk = targetSdkVersion
+  compileSdk = 30
```

Comment:
I noticed Crashlytics uses 33. I am not sure if it matters, but maybe we could use 33 here too so there are only 2 different versions, not 3, when building with other Firebase SDKs.